shims/spark322/src/main/resources/META-INF/services/io.glutenproject.sql.shims.SparkShimProvider

```diff
@@ -1 +0,0 @@
-io.glutenproject.sql.shims.spark322.SparkShimProvider
```

Comment:
required for shim layer

Reply:
rename spark322 module to spark32, see the code above : shims/spark32/src/main/resources/META-INF/services/io.glutenproject.sql.shims.SparkShimProvider